src/Client/KeenIOClient.php

```diff
@@ -7,6 +7,8 @@
 use GuzzleHttp\Client;
 use KeenIO\Exception\RuntimeException;
 
+define('VERSION', '2.5.7');
```

Comment:
please don't define a global constant. This is likely to conflict with any library out there, as the name is ultra generic.

You should either use a class constant or a namespaced constant